### DISCUSSION TITLE
Incremental compilation

### DISCUSSION THREAD

**Discussion by Ruttie2006** [Frogging-Family/linux-tkg#1023] [Q&A] [answered]
Is it possible to use incremental compilation (using intermediary files from previous build) with this repo?
Having to rebuild the kernel all the way from scratch takes a long time, and I though incremental compilation might speed it up.

**Comment by ptr1337** [comment, marked as answer]
Just use ccache

**Reply by Ruttie2006** [reply]
The arch linux wiki states for ccache:
> ccache is effective only when compiling exactly identical sources. (More exactly, preprocessed sources.)

and

> In the Gentoo Linux community, a source based distribution, ccache has been notorious for its placebo effect

Is it truly effective in this usecase?

**Reply by ptr1337** [reply]
Well, it depends how much the kernel source changes. if you disable debug in the kernel config, then it should hit most time

**Reply by Ruttie2006** [reply]
How would I 'disable debug in the kernel config'?
Do you simply mean setting `_debugdisable="true"` in the customization.cfg?

**Reply by ptr1337** [reply]
yes
